File: backend/src/main/java/com/smartcampus/backend/dto/weather/DailyAirQualityDTO.java
package com.smartcampus.backend.dto.weather;

import lombok.*;

import java.time.LocalDate;

@Getter @Setter @NoArgsConstructor @AllArgsConstructor @Builder
public class DailyAirQualityDTO {
    private LocalDate date;
    private Double pm2_5Avg;
    private Double pm10Avg;
    private Double ozoneAvg;
    private Double nitrogenDioxideAvg;
    private Integer europeanAqiMax; // pire valeur de la journee -> sert a la conversion icone/signification
    private String category;
    private String description;
    private String icon;
}

File: frontend/app/src/main/java/com/ObservatoireCampus/mobile/model/weather/DailyAirQualityDto.kt
package com.ObservatoireCampus.mobile.model.weather

data class DailyAirQualityDto(
    val date: String,
    val pm2_5Avg: Double?,
    val pm10Avg: Double?,
    val ozoneAvg: Double?,
    val nitrogenDioxideAvg: Double?,
    val europeanAqiMax: Int?,
    val category: String?,
    val description: String?,
    val icon: String?
)